packages/core-components/src/lib/atoms/inputs/button-group/ButtonGroupItem.svelte

```diff
@@ -17,7 +17,7 @@
 		? 'z-10 border-gray-200 bg-gray-100 text-blue-700'
 		: 'z-0 bg-white text-gray-900 border-gray-200'}
                                 "
-	on:click={() => update({ label, value })}
+	on:click={() => update({ valueLabel, value })}
```

Comment:
Why is `valueLabel` passed up? Seems like it's just ignored anyways in `ButtonGroup` so not sure what the point is

Reply:
Pending some internal discussion the label might be an exposed property of the value, just didn't rip out until that's resolved